docs/site/index/process.md

```diff
@@ -8,7 +8,7 @@ title: CLDR Process
 
 This document describes the Unicode CLDR Technical Committee's process for data collection, resolution, public feedback and release.
 
-- The process is designed to be light-weight; in particular, the meetings are frequent, short, and informal. Most of the work is by email or phone, with a database recording requested changes (See [Requesting Changes](/requesting_changes)).
+- The process is designed to be light-weight; in particular, the meetings are frequent, short, and informal. Most of the work is by email or virtual meeting, with a database recording requested changes (See [Requesting Changes](/requesting_changes)).
```

Comment:
I would rephrase the second sentence to say "Most of the discussion happens over email or in virtual meetings," since the work is not done during meetings?

Reply:
Also "See" in the parenthesis should not be capitalized as it is part of the enveloping sentence.

Reply:
reasonable changes